cfgov/jinja2/v1/_includes/molecules/big-checkbox.html

```diff
@@ -0,0 +1,71 @@
+{# ==========================================================================
+
+   big_checkbox.render()
+
+   ==========================================================================
+
+   Description: Builds checkboxes fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:     Name of the field.
+
+   settings.fields:     An array of all the checkbox labels.
+
+   options (optional):
+
+   options.required:    Whether the field is required. Defaults to false.
+
+   options.disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro render(settings, options) -%}
+    {% from 'macros/util/format/url.html' import slugify as slugify %}
+    {% set id_slug = slugify( settings.legend ) %}
+    <fieldset class="u-reset">
+        <div class="form-l_col
+                    form-l_col-1
+                    u-mb0">
+            <legend class="form-label-header">
+                {{ settings.legend }}
+                {%- if options.required -%}
+                    * <span class="u-visually-hidden">Required</span>
+                {%- endif -%}
+            </legend>
+        </div>
+        {% for field in settings.fields %}
+            {% if loop.length == 2 %}
+                <div class="m-big-checkbox m-big-checkbox__1-2">
```

Comment:
Shouldn't the  `m-big-checkbox` class be on the `fieldset`? If the molecule is always going to be encased within other markup then it should be on the highest level DOM element.